exercises.ipynb

```diff
@@ -63,7 +60,21 @@
     "    # Figure out how to select the text of the title\n",
     "    # of each legoset. A title should look like: \n",
     "    # \"10252: Volkswagen Beetle\"\n",
-    "    pass\n",
+    "    \n",
+    "    divisions = soup.find_all(\"div\", class_ = \"meta\")\n",
+    "    lista = []\n",
+    "    for division in divisions:\n",
+    "        title = division.h1.a.text\n",
+    "        lista.append(title)\n",
+    "    \n",
+    "    if not soup.select(\"li.next a\"):\n",
```

Comment:
This is a bit convoluted, don't run soup.select twice with the same arguments, and why use the negative first?